pallets/disputes/src/impls.rs

```diff
@@ -0,0 +1,25 @@
+use crate::*;
+use frame_support::BoundedVec;
+use sp_runtime::DispatchError;
+
+use crate::pallet::{AccountIdOf, Config, Dispute};
+use traits::DisputeRaiser;
+
+impl<T: Config> DisputeRaiser<AccountIdOf<T>> for Pallet<T> {
+    type DisputeKey = T::DisputeKey;
+    type SpecificId = T::SpecificId;
+    type MaxReasonLength = <T as Config>::MaxReasonLength;
+    type MaxJurySize = <T as Config>::MaxJurySize;
+    type MaxSpecifics = <T as Config>::MaxSpecifics;
+
+    /// Public interface for Dispute::new()
+    fn raise_dispute(
+        dispute_key: Self::DisputeKey,
+        raised_by: AccountIdOf<T>,
+        jury: BoundedVec<AccountIdOf<T>, Self::MaxJurySize>,
```

Comment:
are we defining the jury when we raise the dispute?

Reply:
but then this extrinsic can be called by anyone right? so i can define my own jury?

Reply:
ah wait this is the trait, disregard